x-pack/test/security_solution_endpoint/apps/endpoint/resolver.ts

```diff
@@ -14,7 +14,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const queryBar = getService('queryBar');
 
   // FLAKY: https://github.com/elastic/kibana/issues/77835
-  describe.skip('Endpoint Event Resolver', function () {
+  describe('Endpoint Event Resolver', function () {
```

Comment:
Can you also remove the `// FLAKY` comment if this is fixed now

Reply:
missed this. @bkimmel do you mind adding this change in your PR? Don't want this to rerun the build...